src/services/functionAppService.ts

```diff
@@ -43,63 +44,45 @@ export class FunctionAppService extends BaseService {
     return response.data.value;
   }
 
-  public async deleteFunction(functionName) {
```

Comment:
We don't know if we will need `syncTriggers` or `deleteFucntion` in some other workflows.  Let's keep these methods for now.